src/api/Shrooms.Domain/Services/FilterPresets/FilterPresetService.cs

```diff
@@ -153,11 +153,11 @@ private IQueryable<FilterDto> GetFiltersQueryByFilterType(FilterType filterType)
             };
         }
 
-        private async Task<IEnumerable<FilterPresetDto>> DeleteAsync(ManageFilterPresetDto manageFilterPresetDto)
+        private async Task<IList<FilterPresetDto>> DeleteAsync(ManageFilterPresetDto manageFilterPresetDto)
```

Comment:
Could you explain why using IList<> is better than IEnumerable<>? I know that to get data with EF you would call .ToList(), but here I don't do that.

Reply:
Because it's a private method and where it's used it's clear that it will have to do `ToList()` at least once, but maybe more.
Therefore very time it will have to enumerate.
In such cases it's better to convert explicitly to list once.